polling_stations/apps/pollingstations/admin.py

```diff
@@ -1 +1,65 @@
-# don't enable /admin for this model
+from django.contrib import admin
+from django.contrib.gis.db import models
+from django.contrib.gis.forms import OSMWidget
+from pollingstations.models import PollingStation, VisibilityChoices
+
+
+class PollingStationAdmin(admin.ModelAdmin):
+    actions = ["unpublish", "publish"]
+
+    search_fields = [
+        "address",
+        "postcode",
+        "council__name",
+        "council__council_id",
+        "internal_council_id",
+    ]
+
+    # Filter on published status or council name
+    list_filter = ["visibility", "council__name"]
+
+    # Human-readable name for list view
+    list_display = ["display_name"]
```

Comment:
Could council and visibility by useful here?

Reply:
Currently:
```python
    @admin.display(description="Station Display Name")
    def display_name(self, obj):
        name = obj.address.split("\n")[0]
        council_name = obj.council.name
        internal_council_id = obj.internal_council_id
        return f"{name} - {council_name} ({internal_council_id})"
```

I'll make it:
```python
return f"{name} - {council_name} ({internal_council_id}) ({visibility})"
```

Reply:
So adding items to the list creates columns in the table. 

It's a minor thing I know, so do ignore if you like, but I think it's more readable to have each field in a column. At least 

`list_display = ["display_name", "internal_council_id", "visibility"]` is a quick win, you'd have to make `council_name` another `@admin.display` method.

Reply:
Neat! I hadn't clocked how to make columns.